migrate_cleanup.py

```diff
@@ -0,0 +1,32 @@
+import csv
```

Comment:
The file was used in migration and removed in https://github.com/Samsung/CredData/commit/5046987ceef9916c9afbacf721103942d3c99104
It is useless now. Let it be kept in git history only.